### ISSUE TITLE
Release the package through the public ROS 2 buildfarm

### ISSUE THREAD

**Issue by juhannc** [3473f/ros2autodoc#5]
[Releasing the package](https://docs.ros.org/en/foxy/How-To-Guides/Releasing/Releasing-a-Package.html) to the ROS 2 universe will severely increase its impact.

The first step would probably be to move the repo into its own GitHub group and [file to be a new release team for ROS 2](https://docs.ros.org/en/foxy/How-To-Guides/Releasing/Release-Team-Repository.html#start-a-new-release-team). Once that is done it should be rather easy to follow the necessary steps for a [first release](https://docs.ros.org/en/foxy/How-To-Guides/Releasing/First-Time-Release.html) of the package.

There are also some GitHub actions for ROS (2) to help with the development:

- <https://github.com/marketplace/actions/ros-2-ci-action>
- <https://discourse.ros.org/t/github-actions-for-ros-and-ros-2-ci-beta/12755>
- <https://github.com/ros-industrial/industrial_ci#for-github-actions>

**Comment by juhannc** [comment]
I realized you already applied for a release team at https://github.com/ros2-gbp/ros2-gbp-github-org/issues/202

**Comment by 3473f** [comment]
> I realized you already applied for a release team at [ros2-gbp/ros2-gbp-github-org#202](https://github.com/ros2-gbp/ros2-gbp-github-org/issues/202)

Yes! however, you are more than welcome to work on the Github Actions pipeline and submit a pull request.
